app/templates/search/_services_summary.html

```diff
@@ -2,4 +2,20 @@
   <p class="search-summary">
     {{ summary }}
   </p>
-</div>
+
+  {% if total == 0 %}
+
+  <div class="search-suggestion">
```

Comment:
Looks like this doesn't capture any CSS and is appearing noticeably larger than other text on the page. I think on the prototype they're the same size, or at least _much_ more similar. We should probably use core-14/core-16 in new CCS under `app/assets/scss/_search-page.scss`. Add the core-14/core-16 includes from `scss/toolkit/search/_search-summary.scss`.

<img width="1008" alt="screen shot 2017-07-19 at 14 16 40" src="https://user-images.githubusercontent.com/2920760/28368665-0fb4fffe-6c8d-11e7-967d-6b47665c33cd.png">